spago.yaml

```diff
@@ -62,3 +63,136 @@ workspace:
       git: https://github.com/purescript/registry-dev.git
       ref: 6a803c37577af368caa221a2a06d6be2079d32da
       subdir: foreign
+    css:
```

Comment:
These are transplants from the docs-search's extra dependencies.

Reply:
Some of these seem to be pointing at the latest release (which we are already bringing in via the package set), so can be removed. This is the case with e.g. `css` and `optparse`, but I'm sure there are others